src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -779,7 +779,28 @@ public IEnumerable<Element> GetJoinedElements()
                 .ToList();
         }
 
-
+        /// <summary>
+        /// Switch the order of how two elements are geometry is joined. 
+        /// Note that changing this in 3D will also affect your 2D views.
+        /// </summary>
+        /// <param name="cuttingElement">The element that should be cutting the other element</param>
+        /// <param name="otherElement">The other element that are being cut by the cuttingElement</param>
+        /// <returns>Joined elements</returns>
+        public static IEnumerable<Element> SwitchGeometryJoinOrder(Element cuttingElement, Element otherElement)
+        {
+            if (!JoinGeometryUtils.AreElementsJoined(Document, cuttingElement.InternalElement, otherElement.InternalElement))
+            {
+                return new List<Element>();
+            }
+            if (JoinGeometryUtils.IsCuttingElementInJoin(Document, cuttingElement.InternalElement, otherElement.InternalElement))
+            {
+                return new List<Element>() { cuttingElement, otherElement };
```

Comment:
i assume this means the `cutting` element needs to be joined to the `other` element ? throw as above if not.

Reply:
or is this the bit handling that they are already in the prescribed order ?

Reply:
yes this determines whether the first of two joined elements is cutting the second element.